### PR TITLE
feat: Add initial action

### PR DESCRIPTION
**Pull request by Phylu**
Implement the initial action to trigger a Crashtest Security Scan and download the JUnit XML Report.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: 'Crashtest Security Action'
+description: 'A Github Action for Running a Crashtest Security scan to perform Dynamic Application Security Testing (DAST).'
+inputs:
+  crashtest-webhook:
+    description: 'Webhook Secret of the Crashtest Security Scan Target'
+    required: true
+  pull-report:
+    description: 'Flag whether the report should be downloaded as JUnit XML file'
+    required: false
+    default: 'false'
```
Comment (neotreat) [inline action.yml:10]: The default is intentionally a string value?

Comment (Phylu) [reply action.yml:10]: Yes. I was not able to add this as a boolean. It seems that only string values are supported here.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        // Check if the scan was correctly started
+        if (!scanId) {
+            core.setFailed(`Could not start Scan for Webhook ${crashtestWebhook}.`);
+            return
+        }
+
+        console.log(`Started Scan for Webhook ${crashtestWebhook}. Scan ID is ${scanId}.`)
+
+        // Check if the action should wait for the report and download it
+        if (pullReport == 'false') {
```
Comment (neotreat) [inline index.js]: When the variable needs to be a string I'd use the typesafe comparison here `===`

File: index.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            console.log(`Skipping the download of the scan report as pull-report='${pullReport}'.`);
+            return
+        }
+
+        // Wait until the scan has finished
+        while (status <= 101) {
+            console.log(`Scan Status currently is ${status} (101 = Running)`);
+
+            // Only poll every minute
+            await wait(60000);
```
Comment (neotreat) [inline index.js]: The wait interval might be better located in constant.

Comment (Phylu) [reply index.js]: Done.

File: index.js (hunk 2 above)
Comment (Phylu) [reply index.js]: Done.

**Review by neotreat — APPROVED** [review]

**Pull request merged**